src/modules/editor/selectors.ts

```diff
@@ -5,3 +5,4 @@ export const getEditorMode = (state: RootState) => getState(state).mode
 export const isSidebarOpen = (state: RootState) => getState(state).sidebar
 export const isPreviewing = (state: RootState) => getState(state).preview
 export const getSelectedEntityId = (state: RootState) => getState(state).selectedEntityId
+export const getReadyState = (state: RootState) => !getState(state).isReady
```

Comment:
I find this naming confusing. It's circumstance but having a `getState(state).isReady` that returns the `ReadyState` is too many states for me.
Plus, we're trying to append `is` to boolean functions, so do you think we can rename it to `isReady`?